### DISCUSSION TITLE
Zz Martin's Configuration

### DISCUSSION THREAD

**Discussion by cagnulein** [cagnulein/qdomyos-zwift#2824] [General]
@mrjcleaver i suggest to post here the wiki page https://github.com/cagnulein/qdomyos-zwift/wiki/Zz-Martin's-Configuration and delete it from there.

Thanks

**Comment by mrjcleaver** [comment]
Martin's set-up.

* Echelon EX5
* QZ on Huawei Android 7 (on the tablet holder)
* QZ Companion on Samsung Galaxy Watch 7
* QZ AI on my phone (Samsung Galaxy 22 Ultra)
* Zwift on 4th gen Apple TV

* Samsung Health
* Google Fit

No uploads yet, though I intend to use:
* Strava or Map My Ride

- QZ controlled resistance works, but Zwift doesn't control it yet (perhaps because I'm a beginner in Zwift).
- No gear shifting (feature in beta that the resistance knob can be used for gear shifting but I don't yet have access to the beta)
